best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
This sounds like it's changing ES6 to our default in projhects (over coffeescript). Is that what we've determined?